package.json

```diff
@@ -124,7 +124,6 @@
     "@storybook/manager-webpack5": "6.5.15",
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27380&commitId=3ef00ddb58cb0d2dbce0fe56de99fa66fdb52ef2&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1258 screenshots unchanged, 0 screenshots with different dimensions and 2 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown.Dropdown Example Teams.Focuses first item.chromium.png|1941| Changed|
|Dropdown.Dropdown Example.Focuses first item.chromium.png|1965| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>